docs/workflow-controller-configmap.yaml

```diff
@@ -1,3 +1,6 @@
+#######
+# NEW URL FOR THIS FILE: https://argo-workflows.readthedocs.io/en/latest/workflow-controller-configmap.yaml
+#######
```

Comment:
This was supposed to be only on the `gh-pages` branch since we couldn't redirect this file. But this PR was to `main`. This shouldn't be on `main`

Reply:
Fixed in #12654